docs/templates.md

```diff
@@ -1,296 +1,7 @@
 # Templates
 
-Templates are one of the ways to create files that act as interfaces for your style dictionary. For example, you want to be able to
-use your style dictionary in Android. You can use the `android/colors` template which will create an Anroid resource XML file from
-your style dictionary. You can define custom templates with the [`registerTemplate`](api.md#registertemplate).
+Templates are deprecated in favor of [Formats](formats.md) and will be removed in Style Dictionary 3.0. If you want to create a custom template, create a [Format that uses templating](formats.md).
```

Comment:
I don't remember if this change is going in the 3.0 release or before. In case, "will be removed in Style Dictionary 3.0" => "will be removed in the future".

Reply:
Thanks, will change.